test/ContinuousIntegration/Common/AzurePipelinesLinuxPython_DownloadCache.yml

```
@@ -3,12 +3,12 @@ steps:
 - script: |
     export ITK_PACKAGE_VERSION=$(ITKPythonGitTag)
     curl https://data.kitware.com/api/v1/file/592dd8068d777f16d01e1a92/download -o zstd-1.2.0-linux.tar.gz
-    gunzip -d zstd-1.2.0-linux.tar.gz
-    tar xf zstd-1.2.0-linux.tar
+    tar -xfz zstd-1.2.0-linux.tar.gz
```

```suggestion
    tar -xzf zstd-1.2.0-linux.tar.gz
```
The name of the tar file should follow the -f option. Otherwise "z" is interpreted as the file name: error [here](https://dev.azure.com/simonrit/RTK/_build/results?buildId=189&view=logs&j=eafb87dd-1923-5b11-0b2b-e4ed58eef8aa&t=5551fbe7-80f1-523b-9b5c-d3ebaa80b08e&l=18).

---

Oops. Thanks!